client/my-sites/earn/main.tsx

```diff
@@ -69,16 +69,19 @@ const EarningsMain = ( { section, query, path }: EarningsMainProps ) => {
 		return tabs;
 	};
 
-	const getSelectedText = () => {
-		const selected = find( getFilters(), { path: path } );
+	const getAdSelectedText = () => {
+		const selected = find( getAdTabs(), { path: path } );
 		if ( selected ) {
 			return selected.title;
 		}
 
 		return '';
 	};
 
-	const getComponent = ( currentSection: string ) => {
+	const isAdSection = ( currentSection: string | undefined ) =>
+		currentSection && currentSection.includes( 'ads' );
```

Comment:
`startsWith( 'ads-' )` feels more specific but no biggie either way.

Reply:
Updated this.